addons/message_update_v2.17/enable.js

```diff
@@ -6,6 +6,16 @@
     api.addon.date = (api.settings.updateTime.getTime() / 1000);
   }
 
+
+  // Macos v2.16.0 requires a web-based update.
+  if (api.env.platform === 'macos' && api.env.versionString === '2.16.0') {
+    api.addon.setTitle(
+        'message.message_update_v2.17.block.extra_1_217',
+        'Download the new Mozilla VPN');
```

Comment:
Doesn't this 'Download the new Mozilla VPN' string need to be localized?

Reply:
It's translated via the manifest [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/b65a21c4761369e790b275fc85a2a28e6eec85dc/addons/message_update_v2.17/manifest.json#L58) (also see the comment at the top of `enable.js`)